lib/src/components/Text/docs/index.mdx

```diff
@@ -17,6 +17,12 @@ Set the number of lines you want to display with `lines`. Your text will be disp
 
 <div data-playground="truncation.tsx" data-component="Text"></div>
 
+### wordBreak
+
+Set whether line breaks appear wherever the text would otherwise overflow its content box with the `wordBreak` property. Default is `break-word`.
+
+<div data-playground="wordBreak.tsx" data-component="Text"></div>
+
```

Comment:
Hum au final je pense qu'on peut virer cet exemple car c'est du css pur, donc on le met par défault faudrait juste le dire plus haut ?

Reply:
Yes thanks

It's now documented in the Props section of the Text component page